doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -105,10 +62,235 @@ Rather than explaining the same topic multiple times in multiple locations, modu
+
+## Grammar
+* Give preference to present tense over past and future tenses.
+* Give preference to simple verb forms over perfect and continuous forms.
+* Use call-to-action approach instead of simple statements to help user find the information.
+  
+  **_Good example_**  
+    Click the **Src** tab below to see the source code used to generate the table.
+  
+  **_Bad example_**  
+    Below is the **Src** tab that contains the source code used to generate the table.
+* Avoid using multiple-word verbs, use one-word verbs when you can.
+* Always provide a direct object after a transitive verb, otherwise use an intransitive verb.
+  
+  **_Good example_**  
+    Your application cannot connect to the server.  
+    Installation process will stop.
+  
+  **_Bad example_**  
+    Your application cannot connect.  
+    Installation process will stop running.
+* Avoid using neutral verbs such as _do_, _make_, _have_, etc. when you can use a preciser verb to denote the action.
+  
+  **_Good example_**  
+    Connect to the cloud service.  
+    Update the application regularly.
+  
+  **_Bad example_**  
+    Make a connection to the cloud service.  
+    Do regular application updates.
+* Use imperative mood for procedures, formulate your instructions as giving commands to the user.
+  
+  **_Example_**  
+    Click the **Management** button, enter a new IP in the **Kaa host/IP** block and click **Update**.
+* Use indicative mood to provide facts, questions, assertions, or explanations.
+  
+  **_Example_**  
+    Kaa endpoint connects to your Kaa Sandbox by using the address built into the SDK.
+* Do not use adverbs that do not contribute to the meaning of a sentence.
+  
+  **_Good example_**  
+    The server sends a request.
+  
+  **_Bad example_**  
+    The server quickly sends a request.
+* Reduce ambiguity be providing the right context.
+Some verbs and nouns look the same in writing, for example _access_, _click_, _process_, etc., which can be ambiguous to the user.
+Make sure such words appear in the context that gives them specific meaning.
+* Use second person (pronoun _you_) to support a friendly tone and relation to the user.
+* Use first person in special cases only.
+For example, when you write from the user's point of view.
+* Avoid using more than 3 nouns in a noun string as it may be confusing or ambiguous even for native English speakers.
+  
+  **_Good example_**  
+    This application demonstrates functionality of the Kaa Configuration extension.  
+    Configuration service for client application.
+  
+  **_Bad example_**  
+    This application demonstrates Kaa Configuration extension functionality.  
+    Client application configuration service.
+
+## Text formatting
+* Use **bold** for:
+  * Technical terms that are not in the Glossary, on their first mention.
+  * Titles of windows and dialog boxes.
+  * Names of UI elements, commands, attributes, constants, methods, fields, predefined classes, databases, events.
```

Comment:
As discussed, we leave these in bold. Also, users often seek for particular instructions on how to do something, so bold makes sense here.